httomo/yaml_checker.py

```diff
@@ -325,3 +348,16 @@ def validate_yaml_config(
     )
     _print_with_colour(end_str, colour=Colour.BVIOLET)
     return True
+
+
+class UniqueKeyLoader(yaml.SafeLoader):
```

Comment:
For clarity and background, this class is building on the yaml.SafeLoader and is used from a post at https://stackoverflow.com/questions/33490870/parsing-yaml-in-python-detect-duplicated-keys